### PR TITLE
Winui3

### PR DESCRIPTION
**Pull request by PureWeen**
Add WinUI3 samples project

- WinUI3 doesn't work with dotnet tooling so all the WinUI3 project bits are setup as a separate isolated solution

### CODE REVIEW

File: HelloWinUI3/HelloMaui/Application.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	public class Application : Microsoft.Maui.Controls.Application
+	{
+		public Application(IServiceProvider services)
+		{
+			Services = services;
+		}
+
+		public IServiceProvider Services { get; }
+
+        public override IWindow CreateWindow(IActivationState activationState)
```
Comment (jonathanpeppers) [inline HelloWinUI3/HelloMaui/Application.cs:22]: The spacing isn’t consistent, tabs vs spaces? Do we need an .editorconfig file in this repo?

File: HelloWinUI3/HelloWinUI3.sln [hunk 2]
```diff
@@ -0,0 +1,81 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (jonathanpeppers) [inline HelloWinUI3/HelloWinUI3.sln:2]: Can you add a step to build this solution on the Windows job? Use the MSBuild yaml task, I think it would work.

Comment (PureWeen) [reply HelloWinUI3/HelloWinUI3.sln:2]: Yea... Once there's a published nuget I can use then I'll get this updated

**Review by amono — APPROVED** [review]

File: HelloWinUI3/HelloWinUI3.sln (hunk 2 above)
Comment (PureWeen) [reply HelloWinUI3/HelloWinUI3.sln:2]: all setup

**Comment by Harrybombe** [comment]
Restore

**Review by Harrybombe — APPROVED** [review]
<solution>
    <add key="disableSourceControlIntegration" value="true" />
</solution>

**Review by Redth — APPROVED** [review]

**Pull request merged**